sdk/core/Azure.Core/src/MessageClassifier.cs

```diff
@@ -6,6 +6,10 @@
 namespace Azure.Core
 {
     /// <summary>
+    /// A type that analyzes an HTTP message and determines if the response it holds
+    /// should be treated as an error response. A classifier of this type may use information
+    /// from the request, the response, or other message property to decide
+    /// whether and how to classify the message.
```

Comment:
The wording of my suggestion may not be ideal, but I was thinking it makes sense to explicitly define the behavior of TryClassify semantics somewhere.

```suggestion
    /// A type that analyzes an HTTP message and determines if the response it holds
    /// should be treated as an error response. A classifier of this type may use any information
    /// available on the <see cref="HttpMessage" /> to decide whether and how to classify the message
    /// through TryClassify semantics. TryClassify semantics allow for multiple classifiers to be chained together 
    /// where a false result from TryClassify indicates that the classifier has no opinion on whether the 
    /// response holds an error. A true result indicates that it has an opinion and that any further classifiers
    /// in the chain will not be consulted.
```

Reply:
Ah, I meant to do that in the docstrings for TryClassify, but I clearly forgot to add those docstrings!